src/Inventory.zig

```diff
@@ -1731,12 +1739,14 @@ const TypeEnum = enum(u8) {
 	creative = 1,
 	crafting = 2,
 	workbench = 3,
+	blockInventory = 4,
 };
 const Type = union(TypeEnum) {
 	normal: void,
 	creative: void,
 	crafting: void,
 	workbench: *const main.items.ToolType,
+	blockInventory: void,
```

Comment:
You could just use `normal` here for now. You don't need to (and should not) add a new type, unless you want to implement special behavior on item interaction (such as removing ingredients on crafting).